compiler/testData/codegen/box/controlStructures/forInArray/forInArraySpecializedToUntil.kt

```diff
@@ -0,0 +1,143 @@
+// WITH_RUNTIME
+
+fun checkByteArray():Boolean {
```

Comment:
here and in other functions below, add space before Boolean return type (`fun checkByteArray(): Boolean`)

Reply:
Done, I reformatted the test